eZ/Publish/Core/Limitation/LanguageLimitationType.php

```diff
@@ -132,6 +137,10 @@ public function evaluate(APILimitationValue $value, APIUserReference $currentUse
             return in_array($object->mainLanguageCode, $value->limitationValues, true);
         }
 
+        if ($object instanceof ContentUpdateStruct) {
+            return in_array($object->initialLanguageCode, $value->limitationValues, true);
```

Comment:
This gave me an idea for yet another test like `\eZ\Publish\API\Repository\Tests\Values\User\Limitation\LanguageLimitationTest::testUserIsAllowedToTranslateContent`

1. Create two editors responsible for `eng-GB` and `eng-US` translations respectively, with the following policies:
    - `content/read`, 
    - `content/versionread`,
    - `content/create` with the Language limitation (either `eng-GB` or `eng-US` for the given editor),
    - `content/edit` with the Language limitation (either `eng-GB` or `eng-US` for the given editor),
    - `content/publish` with the Language limitation (either `eng-GB` or `eng-US` for the given editor),
    - `content/translate` with the Language limitation (either `eng-GB` or `eng-US` for the given editor).

2. Change user reference to `eng-GB` editor.
3. Create content in `eng-GB` only. Publish it.
4. Change user reference to `eng-US` editor.
5. Translate Content into `eng-US`. Publish it.

It's slightly different from what you have right now, because it shifts a responsibility of managing entire translations to specific user.

Based on discussions with @SylvainGuittard, I think this is actually real end-user expectation of this feature.

I'm assuming it will work, because `eng-GB` language code exists in the draft being published.

Trick question: if you have last published `eng-US` version (which technically means that the version has both `eng-US` and `eng-GB` languages, is the `eng-GB` editor able to create another version and publish it?) I hope so :)